packaging/deb-systemd/debian/copyright

```diff
@@ -0,0 +1,19 @@
+This work was packaged for Debian by:
+
+    y_uuki <y_uuki@hatena.ne.jp> on Tue, 11 Mar 2014 11:27:51 +0900
+
+It was downloaded from:
+
+    <https://mackerel.io>
+
+Copyright:
+
+    <Copyright (C) 2014 Hatena>
```

Comment:
I'll change this signature and license in next p-r, so currently I'd like to keep this as is.